modules/deploy/pages/deployment-option/cloud/aws-privatelink.adoc

```diff
@@ -0,0 +1,394 @@
+= Configure AWS PrivateLink for Redpanda Cloud
+:description: Set up AWS PrivateLink to securely access Redpanda Cloud. 
+:page-cloud: true
+
+include::shared:partial$feature-flag.adoc[]
+
+The Redpanda AWS PrivateLink endpoint service to allows secure access to Redpanda Cloud from your own VPC. Consider using the endpoint service if you have multiple VPCs and could benefit from a more simplified network management:
+
+* PrivateLink allows you to have overlapping CIDR ranges in VPC networks.
+* PrivateLink does not limit the number of VPC connections. VPC peering on the other hand is limited to 125 connections (see https://aws.amazon.com/privatelink/faqs/["How scalable is AWS PrivateLink?"]).
+
+After <<get-an-access-token,getting an access token>>, you can <<enable-privatelink-endpoint-service-fo- new-clusters,enable PrivateLink when creating a new cluster>>, or you can <<enable-privatelink-endpoint-service-for-existing-clusters,enable PrivateLink for existing clusters>>.
+
+== Requirements
+
+* Ensure you have `rpk` installed.
+* Your Redpanda cluster and <<create-client-vpc,VPC>> must be in the same region.
+* You need to use the Redpanda Cloud API to enable the Redpanda endpoint service for your clusters. Follow the steps below to <<get-an-access-token,get an access token>>.  
+
+== Get an access token 
+
+. You need the base URL of the Redpanda Cloud API:
++
+[,bash]
+----
+PUBLIC_API_ENDPOINT="https://api.cloud.redpanda.com"
+----
+
+. In the Redpanda Cloud UI, go to **Namespaces** and select the namespace where you want to create a cluster.
++
+Copy the namespace ID (UUID) from the URL in the browser.
++
+[,bash]
+----
+NAMESPACE_ID=<uuid>
+----
+
+. Go to the organization (org) level **Home** in the UI, and navigate to **Clients**. If you don't have an existing client, you can create a new one by clicking **Add client**.
++
+Copy the client ID and secret.
```

Comment:
So are these entries doing the actual copying? Or are you taking the data you copied and adding to these entries?

Reply:
@Feediver1 The latter. The code examples in this doc are for the user to copy, fill in with their own specific values, and then run in the terminal or in a script. Should I add a little blurb or note to make it more clear for readers? And @paulzhang97 let me know if I've misunderstood.

Reply:
It is latter. Adding a note should be helpful.